src/gui/src/settings/options-window.cpp

```diff
@@ -283,6 +287,16 @@ OptionsWindow::OptionsWindow(Profile *profile, ThemeLoader *themeLoader, QWidget
 	ui->comboImagePositionVideoH->setCurrentIndex(positionsH.indexOf(settings->value("imagePositionVideoH", "left").toString()));
 	ui->lineImageBackgroundColor->setText(settings->value("imageBackgroundColor", QString()).toString());
 
+	ui->comboActionButtons->clear();
+	ui->comboActionButtons->addItem("All", ZoomWindow::ButtonVisibility::All);
```

Comment:
If you don't want to use `setItemData`, make sure you wrap all translatable strings with the `tr()` function to make sure they're translatable.